packages/forms/resources/views/components/builder.blade.php

```diff
@@ -126,18 +131,12 @@ class="flex items-center justify-center h-8 w-8 rounded-full border border-gray-
 
         @if ((! $isItemCreationDisabled()) && (blank($getMaxItems()) || ($getMaxItems() > $getItemsCount())))
             <div x-data="{ isCreateButtonDropdownOpen: false }" class="relative flex justify-center">
-                <button
+                <x-filament::button
```

Comment:
We cannot use `filament::button` within the forms package, however a `forms::button` button component is being introduced in #2057

Reply:
Will this be merged soon? Then I'll refactor it already

Reply:
This will be merged first